pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go

```diff
@@ -108,16 +108,27 @@ type CustomizationRules struct {
 // LocalValueRetention holds the scripts for retention.
 // Now only supports Lua.
 type LocalValueRetention struct {
-	// LuaScript holds the Lua script that is used to retain runtime values to the desired specification.
+	// LuaScript holds the Lua script that is used to retain runtime values
+	// to the desired specification.
+	//
 	// The script should implement a function as follows:
 	//     luaScript: >
 	//         function Retain(desiredObj, observedObj)
 	//             desiredObj.spec.fieldFoo = observedObj.spec.fieldFoo
 	//             return desiredObj
 	//         end
 	//
-	// LuaScript only holds the function body part, take the `desiredObj` and `observedObj` as the function
-	// parameters or global variables, and finally returns a retained specification.
+	// The content of the LuaScript needs to be a whole function including both
+	// declaration and implementation.
+	//
+	// The parameters will be supplied by the system:
+	//   - desiredObj: the object represents the configuration to be applied
```

Comment:
In fact, the parameters's name is not fixed. It also accept:
```lua
function Retain(d, o)
    d.spec.fieldFoo = o.spec.fieldFoo
    return d
end
```

The only requirements are:
- function name must be `Retain`
- two input args, delivery desired and observed objects
- one return value, delivery the retained object

Reply:
Yes, I think it's common sense that we don't need to explain more. :)